### ISSUE TITLE
Problem on NextQA Inference

### ISSUE THREAD

**Issue by Franklee95** [Yui010206/SeViLA#18]
Hi,
When I using your [SeViLA localizer on QVHighlights](https://huggingface.co/Shoubin/SeViLA/resolve/main/sevila_pretrained.pth) and Inference on NextQA data.I found all predictions are option1.And output key frames of the localizer are indential( like [0,1,2,3]).So what's the potential reason of this problem?
<img width="1157" alt="image" src="https://github.com/Yui010206/SeViLA/assets/41943056/824a58f5-b6a6-4ddd-8ca7-3bf7d65d41c1">

**Comment by Franklee95** [comment]
I have found the reason! I print the output logits of the t5 model(sevila.py #587)
<img width="400" alt="image" src="https://github.com/Yui010206/SeViLA/assets/41943056/91df4eaa-624c-4e0e-a7c3-aa055b7e9ddd">
I found that all tensors are NaN
<img width="970" alt="image" src="https://github.com/Yui010206/SeViLA/assets/41943056/045c0dfb-1175-4b42-bb81-35dd981fb0d6">

**Comment by Franklee95** [comment]
Thanks god! Finally I solve the problem! The real reason is that my gpu device(V100) cannot support torch.bfloat16.So I change from bfloat16 to float16 , which will lead to out of mixture.Should change from bfloat16 to float32 will get the correct answer

**Comment by yangk-910** [comment]
Hello, how did you change the configuration file? I attempted to run on my own dataset and reported an error after changing the configuration file, as shown in the following figure. Can you tell me how you made the changes?
![image](https://github.com/Yui010206/SeViLA/assets/62458963/45e2b26f-48de-41a5-9cf5-30ce94fcb5b0)
![image](https://github.com/Yui010206/SeViLA/assets/62458963/f78b93f8-e860-492f-af61-48598884fbe2)
